### ISSUE TITLE
Issues with using JSQMessagesViewController classes embedded in SKYKitChat

### ISSUE THREAD

**Issue by cheungpat** [SkygearIO/chat-SDK-iOS#136]
@tensiuyan discovered that the [SkygearChat-iOS-Demo](https://github.com/skygear-demo/SkygearChat-iOS-Demo) is unable to compile with 1.2.0-alpha.1 of SKYKitChat/UI. Compilation problems are related to the use of JSQ classes in the demo project.

It appears that the JSQMessagesViewController library is embedded into SKYKitChat, and so from the Cocoapods point of view, there is no longer a framework with the name `JSQMessagesViewController`, and `import JSQMessagesViewController` in swift demo project no longer works.

The podspec doesn’t specify the header files in JSQMessagesViewController, so swift demo project cannot use the JSQ classes either. (I am going to send a pull request for this.)

If accepting the pull request, the swift demo has to make the following changes in order to use SKYKitChat:

* Remove `JSQMessagesViewController` from `Podfile` because the classes are already embedded in SKYKitChat
* Replace all instances of `import JSQMessagesViewController` with `import SKYKitChat` (if the latter one is not already imported)

In other words, people who need to use JSQ classes must not include `JSQMessagesViewController` as a dependency in their `Podfile` and should `import SKYKitChat` in swift code.

This issue serves to discuss the recommended way for developer who need to use JSQ classes in their chat app.
